CMP1903_A1_2324/Testing.cs

```diff
@@ -1,20 +1,38 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace CMP1903_A1_2324
 {
-    internal class Testing
-    {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
+    internal class Testing { // testing the game and die class
+        
```

Comment:
If you added a constructor here you wouldn't have to call the two methods in the program class!